docs/docs/zh/guide/task/switch.md

```diff
@@ -13,25 +15,34 @@ Switch是一个条件判断节点，依据[全局变量](../parameter/global.md)
 - 描述信息：描述该节点的功能。
 - 任务优先级：worker线程数不足时，根据优先级从高到低依次执行，优先级一样时根据先进先出原则执行。
```

Comment:
```suggestion
- 任务优先级：worker 线程数不足时，根据优先级从高到低依次执行，优先级一样时根据先进先出原则执行。
```
It is recommended to add spaces between Chinese characters, numbers, and English. The same goes for the content behind it.